auth/src/main/res/values/colors.xml

```diff
@@ -3,7 +3,7 @@
     <!-- Eventually, we should remove these so the developer knows they have to style FirebaseUI -->
     <color name="colorPrimary" tools:ignore="ResourceName">#3F51B5</color>
     <color name="colorPrimaryDark" tools:ignore="ResourceName">#303F9F</color>
-    <color name="colorAccent" tools:ignore="ResourceName">#FF4081</color>
+    <color name="colorSecondary" tools:ignore="ResourceName">#FF4081</color>
```

Comment:
With the breaking changes of this PR, it might be a good time to "Eventually, we should remove these so the developer knows they have to style FirebaseUI"